Source/Core/Core/Config/MainSettings.h

```diff
@@ -232,12 +232,12 @@ extern const Info<std::string> MAIN_IMGUI_FONT;
 
 enum class OutlineRes
 {
-  None,
+  Off,
   Thin,
   Thick,
   Full
 };
-extern const Info<OutlineRes> MAIN_IMGUI_OUTLINE_RES;
+extern const Info<Config::OutlineRes> MAIN_IMGUI_OUTLINE_RES;
```

Comment:
Why the namespace change? Is it consistent with the rest of the file?

Reply:
Linux was complaining about it, but I still don't know why

Reply:
Issue #58 has the error that occurs without it

Reply:
Also it's not consistent with the rest of the file which is why I'm confused about the error